src/components/shared/Header.astro

```diff
@@ -14,7 +14,7 @@
     </div>
     <ul class="menu">
       <li class="menu-item arrow">
-        <a><span>P2P Media Loader</span></a>
+        <a href="/"><span>P2P Media Loader</span></a>
```

Comment:
Why `Contact` was changed to `<div class="anchor"><span>Contact</span></div>` but this one leave as `<a>` with wrong `href`

Reply:
![image](https://github.com/Novage/corporate-site/assets/91938357/ce7a3ede-378c-4974-a691-85dc6f98fc7e)
Those changes were made because of lighthouse. For the first anchor tag, I decided to make href to our index page and for second one , I decided to replace it with div block, since href="/" for contact section wouldn't be logical.
I think that it would be good to replace first uncrawled anchor with div to escape warnings from lighthouse.
[see what google writes about uncrawled links](https://developers.google.com/search/docs/crawling-indexing/links-crawlable?visit_id=638430970339425008-1794813510&rd=1)